README.md

```diff
@@ -1,16 +1,127 @@
-# Title
+# Creating Functions in JavaScript
 
 ## Learning Goals
 
-- SWBAT 1
-- SWBAT 2
+- State that functions are ways to wrap up behavior into a bit of
+code (State the Purpose of Functions)
+- State that JavaScript functions are “First Class Data" (What are
+JavaScript Functions)
+- Demonstrate how to assign a function to a variable (Demonstrate
+Assigning Functions to Variables)
+- Demonstrate how to pass behavior into a function (Demonstrate
+Passing Behaviors into Functions)
 
 ## Introduction
 
-### SWBAT 1
+Sometimes in life, we need to take a series of steps to complete a task. 
+Since life isn't scripted as scripted as a TV show for example, anything
+can happen that alters each step. We need to be able to adjust accordingly.
 
-### SWBAT 2
+## State the Purpose of Functions
+
+Functions are ways to wrap up behavior into a bit of code. Functions are one
+of the key pieces of any programming language. Functions can often apply to 
+everyday things. Think of functions like a list of actions to take to get a
+desired outcome, for example, making recipes from a cookbook.
+
+```js
+   function makeMeASandwich(ingredients) {
+       let sandwich = ingredients
+       return ingredients
+   }
+```
+
+## What are JavaScript Functions
+
+A function is a group of code that can be called upon, sometimes
+by a name. In other words, a function is a way to group together
+some code, give this group a name, and later invoke the code using
+the given name.
+
+Functions in JavaScript are first-class functions. This means that
+we can pass them as values to other functions, like numbers, strings,
+arrays, and objects. They're super useful.
+
+According to the Mozilla developer glossary:
```

Comment:
I'm not sure this definition adds much clarification here. I think it's more confusing than clarifying.